src/partials/footer.html

```diff
@@ -1,93 +1,91 @@
 <footer class="footer-section">
   <div class="container footer-container">
-    <div class="footer-right-side">
-      <div class="footer-image-container">
-        <img
-          class="footer-image"
-          src="../images/footer-img.png"
-          alt="woman"
-          width="349"
-          height="179"
-        />
+    <div class="footer-image-container">
+      <img
+        class="footer-image"
+        src="../images/footer-img.png"
+        alt="woman"
+        width="349"
+        height="179"
+      />
 
-        <a class="footer-logo-container logo-container" href="./index.html">
-          <svg width="16" height="16" class="footer-logo-img">
-            <use href="../images/icons.svg#logo-vector"></use>
-          </svg>
-          <span class="footer-logo-text">FRESH HARVEST BOX</span>
-        </a>
+      <a class="footer-logo-container logo-container" href="./index.html">
+        <svg width="16" height="16" class="footer-logo-img">
+          <use href="../images/icons.svg#logo-vector"></use>
+        </svg>
+        <span class="footer-logo-text">FRESH HARVEST BOX</span>
+      </a>
```

Comment:
aici ar fi mai bine ca a sa cuprinda un div.logo-container. pentru ca o sa trebuiasca sa ai un gap intre icon si span si ar fi mult mai usor daca ai face asa. Deci link ul sa cuprinda divul.logo-container si in interiorul lui sa fie svg ul si span ul